dangerfile.js

```diff
@@ -0,0 +1,144 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+'use strict';
+
+const {markdown, danger} = require('danger');
+const fetch = require('node-fetch');
+
+const {generateResultsArray} = require('./scripts/rollup/stats');
+const currentBuildResults = require('./scripts/rollup/results.json');
+
+/**
+ * Generates a Markdown table
+ * @param {string[]} headers
+ * @param {string[][]} body
+ */
+function generateMDTable(headers, body) {
+  const tableHeaders = [
+    headers.join(' | '),
+    headers.map(() => ' --- ').join(' | '),
+  ];
+
+  const tablebody = body.map(r => r.join(' | '));
+  return tableHeaders.join('\n') + '\n' + tablebody.join('\n');
+}
+
+/**
+ * Generates a user-readable string from a percentage change
+ * @param {string[]} headers
+ */
+function emojiPercent(change) {
+  if (change > 0) {
+    return `:small_red_triangle:+${change}%`;
+  } else if (change <= 0) {
+    return `${change}%`;
+  }
+}
+
+// Grab the results.json before we ran CI via the GH API
+const baseMerge = danger.github.pr.base.sha;
```

Comment:
Are we sure this one is right? Can we log it and see what it's equal to for this PR?